.github/workflows/daily-test.yml

```diff
@@ -19,7 +19,6 @@ jobs:
                     python-version: 3.9
             -   name: Install dependencies
                 run: |
-                    make upgrade-dist-tools
```

Comment:
I deleted this command. It is better to include the dist-tools as dev-dependencies (in the dev-section of Pipfile). This way, they get installed when `make install-requirements` is executed. We don't need a separate command just to install the dist-tools.